docs/source/content/extending.rst

```diff
@@ -0,0 +1,32 @@
+Extending Django Drip
+=====================
+
+Django Drip provides a way for you to extend all the functionality
+Trough abstract classes.
+
+These classes provide all the functions required to make a drip work, but they don't provide any data column.
+Beware tough these classes use the attributes from the concrete drip casses shown in the rest of the docs.
+
+For you to extend them you need to overwite the implementations with your own.
+Since this classes handle themselves mainly trough related models, as long as you maintain the inteface you can extend in any way you want.
```

Comment:
```suggestion
Since this classes handle themselves mainly through related models, as long as you maintain the inteface you can extend in any way you want.
```